src/backend/aspen/workflows/nextstrain_run/run_nextstrain_scheduled.sh

```diff
@@ -30,8 +30,6 @@ TEMPLATE_ARGS=$(jq -c . < "${TEMPLATE_ARGS_FILE}")
 
 workflow_id=$(aspen-cli db create-phylo-run                                                                           \
                   --group-name "${GROUP_NAME}"                                                                               \
-                  --all-group-sequences                                                                                      \
-                  --builds-template-file "/usr/src/app/aspen/workflows/nextstrain_run/builds_templates/${TEMPLATE_FILENAME}" \
```

Comment:
With this change, the template arg [here](https://github.com/chanzuckerberg/aspen/blob/bca255dae34aeaeac42b44e241df9bb345c95675/src/backend/aspen/workflows/nextstrain_run/run_nextstrain_scheduled.sh#L9), [here](https://github.com/chanzuckerberg/aspen/blob/bca255dae34aeaeac42b44e241df9bb345c95675/.happy/terraform/modules/sfn_config/nextstrain.wdl#L12) and [here](https://github.com/chanzuckerberg/aspen/blob/bca255dae34aeaeac42b44e241df9bb345c95675/.happy/terraform/modules/ecs-stack/county_runs.tf#L28) will no longer be needed or used, is that right?

Also I know this will break my fix for Chicago overview, that is ok. I'll add in a fix per yaml builder way 😉

Reply:
yes, template_filename will be 100% ignored. but let's figure out how we can take care of Chicago before we merge this!